packages/components/src/components/bal-navigation/bal-meta-mobile-head/bal-meta-mobile-head.tsx

```diff
@@ -0,0 +1,28 @@
+import { Component, h, ComponentInterface, Host, Prop } from '@stencil/core'
+import { BEM } from '../../../utils/bem'
+
+@Component({
+  tag: 'bal-meta-mobile-head',
+})
+export class MetaMobileHead implements ComponentInterface {
+  /**
+   * aria label for mobile meta navigation
+   */
+  @Prop() ariaLabelMobileHead = ''
+
+  render() {
+    const metaEl = BEM.block('nav').element('metamobile')
+
+    return (
+      <Host class={{ ...metaEl.class(), 'is-hidden-desktop': true }}>
+        <nav role="navigation" aria-label={this.ariaLabelMobileHead}>
+          <slot name="logo" />
```

Comment:
Can the logo be changed or is it always the same?

Reply:
as discussed, this needs to be checked with design as well. btw component itself deleted and integrated in bal-navigation